endpoints/openrtb2/auction.go

```diff
@@ -285,6 +331,15 @@ func (deps *endpointDeps) parseRequest(httpRequest *http.Request) (req *openrtb_
 		}
 	}
 
+	requestJson, reject := deps.hookExecutor.ExecuteEntrypointStage(httpRequest, requestJson)
+	if reject != nil {
+		errs = []error{reject}
+		if err = json.Unmarshal(requestJson, req.BidRequest); err != nil {
+			glog.Errorf("Failed to unmarshal BidRequest during entrypoint rejection: %s", err)
+		}
+		return
+	}
```

Comment:
Why are we trying to unmarshal if the module has issues a rejection?

Reply:
You are right, this makes no sense within this PR, so I've removed it.
In one of the next PRs, we are adding debug info to `BidResponse`, but only if the debug mode enabled, which is determined from `BidRequest`.